uint8 id
uint8 cmd_mode
float32 Pos 
float32 Vel
float32 limit_cur